Fabric.Authorization.AccessControl/src/app/services/fabric-auth-user.service.ts

```diff
@@ -69,6 +69,10 @@ export class FabricAuthUserService extends FabricBaseService {
     subjectId: string,
     roles: IRole[]
   ): Observable<IUser> {
+    if (!roles || roles.length === 0) {
+      return Observable.of(undefined);
```

Comment:
would it be better to return an observable of `undefined`, or an empty observable?  It seems to me that if you do that, the subscriber will have to check the value for `undefined`, right?

Reply:
done